akka-javasdk-maven/akka-javasdk-archetype/src/main/resources/archetype-resources/src/test/java/__packageInPathFormat__/HelloWorldIT.java

```diff
@@ -11,7 +11,7 @@
  * It interacts with the components of the application using a componentClient
  * (already configured and provided automatically through injection).
  */
-public class IntegrationTest extends TestKitSupport {
+public class HelloWorldIT extends TestKitSupport {
```

Comment:
We can keep this as `IntegrationTest` because `IntegrationTest` is excluded from the `mvn test`. Also, `HelloWorldIT` is completely unrelated with the archetype. 

IMO, we should prefer suffixing the test with `IntegrationTest` instead of `IT` in all cases because it makes it more obvious.

I understand that the folks from the fail failsafe plugin had to choose for `IT`, because they don't know where it will be used. So they needed to step out of `*IntegrationTest` to avoid conflicts with the more general `*Test`. 

But our parent pom configures the failsafe plugin and the surefire plugin. So we don't need to favor `IT`.

Then we have:

* `*Test` for unit tests, picked by surefire
* `*IntegratrionTest` and all the variants with `IT` (as originally define by failsafe plugin)

Reply:
Ok, my idea was to follow the convention but I think you make a valid point. It's for sure clearer.